.circleci/config/executors/@executors.yml

```diff
@@ -0,0 +1,25 @@
+go-machine:
```

Comment:
Note to reviewers: This file contains code migrated from `.circleci/config/@config.yml` See [this comment](https://github.com/hashicorp/vault/pull/9653/files#r465849114) (above)